### PR TITLE
feat(admin credential): board improvements

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Credential Request Management Board Improvements

## Why

UI Improvements

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/619

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/assets/locales/de/main.json [hunk 1]
```diff
@@ -1796,7 +1796,17 @@
         "crendentialType": "Type",
         "companyInfo": "Company Info",
         "certificate": "Zertifikat",
-        "document": "Dokumente"
+        "document": "Dokumente",
+        "type": {
+          "TRACEABILITY_FRAMEWORK": "Traceability",
+          "PCF_FRAMEWORK": "Pcf Framework",
```
Comment (oyo) [inline src/assets/locales/de/main.json:1802]: I guess that abbrev. should be spelled `PCF`

File: src/assets/locales/en/main.json [hunk 2]
```diff
@@ -1872,7 +1872,17 @@
         "crendentialType": "Type",
         "companyInfo": "Company Info",
         "certificate": "Certificate",
-        "document": "Document"
+        "document": "Document",
+        "type": {
+          "TRACEABILITY_FRAMEWORK": "Traceability",
+          "PCF_FRAMEWORK": "Pcf Framework",
```
Comment (oyo) [inline src/assets/locales/en/main.json:1878]: I guess that abbrev. should be spelled `PCF`

File: src/components/pages/AdminCredential/AdminCredentialElements.tsx [hunk 3]
```diff
@@ -304,12 +305,23 @@ export default function AdminCredentialElements() {
...
+                children={
+                  <div>
+                    <SettingsBackupRestoreIcon
+                      className="revokeBtn"
+                      onClick={() => {
+                        setOpenRevokeOverlay(true)
+                        setCredentialData(row)
+                      }}
+                    />
+                  </div>
```
Comment (oyo) [inline src/components/pages/AdminCredential/AdminCredentialElements.tsx:323]: do we need that <div> as wrapper here?

**Review by oyo — CHANGES_REQUESTED** [review]
small changes

File: src/components/pages/AdminCredential/AdminCredentialElements.tsx (hunk 3 above)
Comment (nidhigarg-bmw) [reply src/components/pages/AdminCredential/AdminCredentialElements.tsx]: @oyo Made the changes

**Pull request merged**